addons/test_mail/models/test_mail_corner_case_models.py

```diff
@@ -69,7 +69,7 @@ class MailTestMultiCompany(models.Model):
     """ This model can be used in multi company tests"""
     _name = 'mail.test.multi.company'
     _description = "Test Multi Company Mail"
-    _inherit = 'mail.thread'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
```

Comment:
Side note: would create a new model (mail.test.multi.company.alias ?), so that we don't mix features and tests.

Reply:
Done: added specific model for testing activity.